scipy/stats/tests/test_mstats_basic.py

```diff
@@ -961,6 +961,29 @@ def regression_test_9033(self):
         x = np.hstack([np.full(c, i) for i, c in enumerate(counts)])
         assert_equal(mstats.kurtosistest(x)[1] < 0.01, True)
 
+    @pytest.mark.parametrize("test", ["skewtest", "kurtosistest"])
+    @pytest.mark.parametrize("alternative", ["less", "greater"])
+    def test_alternative(self, test, alternative):
```

Comment:
This currently compares the output of these functions against the `stats` versions when there are no masked values. Can you test it with masked values, and for the `stats` version use the [`compressed`](https://numpy.org/doc/stable/reference/generated/numpy.ma.MaskedArray.compressed.html) array?

Even better would be to test the multi-dimensional case, parametrizing over the values of the `axis` argument. But for that, we can wait until gh-13312 gets merged. Then it will be really easy to add multi-dimensional tests for a `mstats` and `nan_policy='omit'` cases.

Reply:
> Can you test it with masked values, and for the `stats` version use the [`compressed`](https://numpy.org/doc/stable/reference/generated/numpy.ma.MaskedArray.compressed.html) array?

Sure!



> Even better would be to test the multi-dimensional case, parametrizing over the values of the `axis` argument. But for that, we can wait until [gh-13312](https://github.com/scipy/scipy/pull/13312) gets merged.

That's a great idea too. I can wait until gh-13312 gets merged. I will keep this in mind until then!